.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,17 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag todo-app:test .
+    - run: docker run --env-file ./.env.test --publish 80:8000 todo-app:test
```

No need to publish any ports here, since the tests are all running within the container
```suggestion
    - run: docker run --env-file ./.env.test todo-app:test
```